alacritty/src/display.rs

```diff
@@ -31,7 +31,7 @@ use parking_lot::MutexGuard;
 #[cfg(not(any(target_os = "macos", windows)))]
 use wayland_client::{Display as WaylandDisplay, EventQueue};
 
-use font::{self, Rasterize};
+use font::{self, Rasterize, set_font_smoothing};
```

Comment:
@chrisduerr I've moved the `fn` to the font crate now. How do we like this?

Reply:
Seems much better, thanks.